src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -7708,7 +7684,7 @@ void SizeVRF(EnergyPlusData &state, int const VRFTUNum)
             state.dataHVACVarRefFlow->VRFTU(VRFTUNum).MaxCoolAirVolFlow = sizingCoolingAirFlow.size(state, TempSize, errorsFound);
 
         } else if (SAFMethod == FlowPerCoolingCapacity) {
-            SizingMethod = CoolingCapacitySizing;
+            SizingMethod = CoolingCapacitySizing; // either this isn't needed or needs to be assigned to EqSizing
```

Comment:
Usually this equivalence would be followed by setting EqSizing so either EqSizing is missing (need to check sizing calls) or this line can be deleted. So left it here as a reminder.

        EqSizing.SizingMethod(SizingMethod) = SAFMethod;